schema/definitions/0.8.0/schema/fmu_results.json

```diff
@@ -1370,7 +1369,31 @@
                         "ssdl"
                     ]
                 }
-            }
+            },
```

Comment:
Might be more readable if this is placed in the definition of `data` further up, somewhere here: https://github.com/equinor/fmu-dataio/blob/df05acabaeb818f37278400a00cfbc8ea654bf77/schema/definitions/0.8.0/schema/fmu_results.json#L177

...rather than at the base of the schema.

Reply:
Is this possible? Since were checking for the value of `class` to conditionally require a property within the `data` object, it will have to be done at a higher level in the schema? Tried moving the `if/then` logic to the `data` definition but the test fails. Any ideas?

Reply:
Aha, I did not consider that `class` is one level down... Then you are probably right.